source/standards/git.html.md.erb

```diff
@@ -131,35 +142,37 @@ branch in the repository history. This advice may be freely ignored for smaller
 local feature branches for which a fast-forward merge will look like any other
 routine development work on `master`.
 
-## Never call `git push -f` without additional arguments
+## Don't use `git push -f`, use `--force-with-lease` instead
 
 Force pushing in git is a subject that attracts all kinds of religious
-battles. This advice is not about the merits of force pushing.
-
-This is about how to use `git push -f` if and when you do use it.
+battles. This advice is not about the merits of force pushing. This is about how
+to use `git push -f` if and when you do use it.
 
 Let's say you're working on a branch, 'foobar', and you decide to force push
 to the remote. So you do this:
 
     $ git push -f
 
-If anyone has committed anything to master[^1] since you last pulled -- and if
-you've been working on the branch for any length of time this is pretty likely
--- you will blow their changes away, because without arguments git will force
-push *all* remote-tracking branches.
+If anyone else has committed anything to your branch since you last pulled, you
+will blow their changes away.
 
-So, if you ever need to force push the 'foobar' branch, please instead do
+So for a bit of safety, if you ever need to force push please instead do:
 
-    $ git push --force-with-lease origin foobar
+  $ git push --force-with-lease
 
 `--force-with-lease` refuses to update a branch unless it is the state that we expect; i.e. nobody has updated the remote branch.
 See: https://developer.atlassian.com/blog/2015/04/force-with-lease/
 
-You can also change git's default behaviour of pushing all tracked branches by
-doing `git config --global push.default upstream`, but you should probably get
-into the habit of typing out your intentions in full when doing a destructive
-operation like force pushing, otherwise disaster will strike when you use
-someone else's differently-configured git, or miss a step when configuring a
-new machine.
+## File content conventions
+
+Where possible, it is a good idea to keep line length to around 80 characters.
```

Comment:
This varies by language community, I think, and is inconsistent with guidance elsewhere, for example https://gds-way.cloudapps.digital/manuals/programming-languages/python/style-guide.html

Reply:
Section removed 👍